Xamarin.Forms.Platform.Android/Renderers/LabelRenderer.cs

```diff
@@ -164,8 +166,10 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateLineHeight();
 			else if (e.PropertyName == Label.MaxLinesProperty.PropertyName)
 				UpdateMaxLines();
-			else if (e.PropertyName == ImageButton.PaddingProperty.PropertyName)
```

Comment:
I noticed this coming in from another commit and although it probably works because of the same string value, it felt wrong.